content/scaling-kubernetes-cluster/scale-nginx-ingress-vertically.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Scale Nginx Ingress pods vertically"
+description: "Procedures to scale Nginx Ingress pods vertically by increasing CPU and memory limits"
+weight: 6
+---
+
+{{% dialog type="warning" icon="fa-info-circle" title="Prerequisites" %}}
+This assumes you've followed the [Geodesic Module Usage]({{< relref "geodesic/module/with-kops.md" >}}) guide which covers all the scaffolding necessary to get started.
+{{% /dialog %}}
+
+Kubernetes Nginx [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) controller is built around the [Kubernetes Ingress resource](https://kubernetes.io/docs/concepts/services-networking/ingress/) 
```

Comment:
We shouldn't copy and paste this 3 times. It makes it hard to maintain. Better to think about information architecture. If we need to copy it, then either the pages should be combined and separated by chapter headings or some other organization considered.